src/scss/06_components/paragraphs/_event-list.scss

```diff
@@ -8,186 +8,122 @@ $event-list-image-mobile-height: 264px;
+.event-form__filters-container {
+  margin-top: $spacing;
 
-  @include breakpoint($breakpoint-l) {
-    align-items: flex-start;
+  .event-form__filter-section-container {
     display: flex;
     flex-wrap: wrap;
+    gap: $spacing-and-half;
+    justify-content: space-between;
+
+    > div {
+      flex-basis: 100%;
+    }
+
+    @include breakpoint($breakpoint-m) {
+      > div {
+        flex-basis: 48%;
+      }
+    }
   }
-}
 
-.event-list__tag {
-  @include font('special.helper_text');
-  background: $color-silver;
-  border-radius: $spacing-and-half;
-  display: inline-block;
-  margin-bottom: $spacing;
-  padding: $tag-vertical-padding $tag-horizontal-padding;
-}
-
-.event__icon {
-  align-content: flex-start;
-  display: flex;
-  margin-right: $spacing-half;
-}
-
-.event__content {
-  @include font('body');
-  display: flex;
-  margin-bottom: $spacing;
-}
+  .event-form__filter-checkbox-container {
+    display: flex;
+    flex-direction: column;
 
-.event-list__indicator-container {
-  align-items: flex-end;
-  display: flex;
-}
+    .hdbt-search__checkbox {
+      margin-top: var(--spacing-2-xs);
+    }
 
-.event-list__event-link-indicator {
-  @include pseudo-icon('link-external', $spacing-and-half);
+    @include breakpoint($breakpoint-m) {
+      flex-direction: row;
+      flex-wrap: wrap;
 
-  @include breakpoint($breakpoint-l) {
-    @include pseudo-icon('link-external', $spacing-triple);
+      @include flex-gap($spacing-and-half);
+    }
   }
 }
 
-.event__content::before {
-  margin-right: $spacing-half;
-}
-
-.event__content--date::before {
-  @include pseudo-icon('calendar-clock', $spacing-and-half);
-}
-
-.event__content--location::before {
-  @include pseudo-icon('location', $spacing-and-half);
-}
-
-.event-list__list-container {
-  margin-bottom: $spacing-double;
-}
-
-.event__lower-container {
-  display: flex;
-  justify-content: space-between;
-
+// Override HDS style for multiple checkboxes
+.event-form__filter-section-container .hdbt-search__checkbox:not(:first-of-type) {
   @include breakpoint($breakpoint-l) {
-    margin-bottom: $spacing-half;
+    margin-top: 0;
   }
 }
 
-.event-list__count {
-  @include font('body');
+.event-form__checkboxes-label {
+  font-weight: $font-weight-medium;
   margin-bottom: $spacing;
 }
 
-.events-list__empty-subtext {
-  margin-bottom: $spacing-and-half;
-}
-
-.event-list__load-more {
-  @include flex-gap($spacing);
-}
-
-.event-list__load-more-button {
-  margin-bottom: $spacing;
+.event-list__submit-button {
+  margin-bottom: $spacing-triple;
+  margin-top: $spacing;
 }
 
-.event-list-spinner {
-  position: relative;
+.event-list__see-all-button {
+  margin: 32px auto 0;
+  text-align: center;
 }
 
-// Use a specific selector here since spinner styles are loaded later than paragraphs
-.event-list-spinner .hds-loading-spinner {
-  margin-bottom: $spacing-triple;
-  margin-top: $spacing-triple;
-  position: relative;
+.event-list__loading-spinner > div {
+  margin: 0 auto;
```

Comment:
Are you sure you want to set the top and bottom margins to 0? If not we can use `margin-inline: auto;`.